sections/antiadblock.txt

```diff
@@ -17,6 +17,9 @@ onvideo.hu###detect + .wrapper
 @@||bitepito.hu/rejtveny/data/ads/adsense.js^$script
 @@||bitepito.hu/rejtveny/data/ads/googleads.js^$script
 @@||meszotar.hu/scripts/advertising.js
+! https://github.com/hufilter/hufilter-dev/issues/363
+myonlineradio.hu##.layer_cont
+myonlineradio.hu##._close.overlay
```

Comment:
@ehrhardt06 what kind of anti adblock is on this website?

Reply:
<details>
<summary>this one</summary>
<br>

![antiadblock2](https://github.com/hufilter/hufilter-dev/assets/137706181/7d787ae5-952b-45fb-90df-7828a8838c73)


</details>

Although now that you ask, maybe I should have put it in the annoyances.txt, but I'm not sure.